saleor/campaign/models.py

```diff
@@ -14,3 +16,23 @@ class Campaign(models.Model):
     seniority = models.CharField(max_length=30)
     education = models.CharField(max_length=40)
     job_function = models.CharField(max_length=255)
+    created = models.DateTimeField(auto_now_add=True)
+
+
+class JobInfo(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    title = models.CharField(max_length=250, blank=True)
+    campaign = models.OneToOneField('campaign.Campaign', related_name='job_info', on_delete=models.CASCADE)
+    industry = models.CharField(max_length=50)
```

Comment:
Makes sense for industry, education, seniority and job function to be part of the JobInfo model. Why are they also present in the Campaign model?